### DISCUSSION TITLE
Import missing references via vim

### DISCUSSION THREAD

**Discussion by KRILLEKS** [JetBrains/ideavim#926] [Q&A] [answered]
What is name of action for importing missing references in file?

**Comment by citizenmatt** [comment, marked as answer]
The best thing to do is to invoke Find Action (shift + shift and tab over) and search for "IdeaVim track action ID". This will show a notification with the last executed action ID. Repeat whatever you did to import references, and as long as it's a normal action, you should get the action ID
